packages/react/src/themes/teams-high-contrast/components/Button/buttonVariables.ts

```diff
@@ -15,6 +16,8 @@ export default (siteVars: any): Partial<ButtonVariables> & ButtonHighContrastVar
     primaryBackgroundColorActive: siteVars.colors.white,
     primaryBackgroundColorFocus: siteVars.accessibleCyan,
 
+    secondaryAltPressedColor: siteVars.colors.black,
```

Comment:
Should this be renamed to `secondaryAltForegroundColor` or something since it's used for more than just `pressed` state?

Reply:
Actually, is this needed or does the color schemes use the right values here? If not, should those be updated or new values added?

Reply:
The color scheme doesn't work with this button for some reason. This is the hover state if I remove all the overrides:
![image](https://user-images.githubusercontent.com/6237759/70576964-10727100-1b5f-11ea-8556-09f42f065987.png)